python/packages/autogen-ext/pyproject.toml

```diff
@@ -56,6 +56,11 @@ redis = [
 grpc = [
     "grpcio~=1.62.0", # TODO: update this once we have a stable version.
 ]
+
+azure-ai-inference = [
+    "azure-ai-inference>=1.0.0b6",
```

Comment:
This should be merged with the `azure` extra, because if you want to use this you need `azure-identity` anyway.